xz/FeatureModel.xml

```diff
@@ -2033,32 +1507,29 @@
       <parent>Size</parent>
       <impliedOptions />
       <excludedOptions />
-      <minValue>1</minValue>
+      <minValue>0</minValue>
       <maxValue>9223372036854775807</maxValue>
       <stepFunction>Size_val + 1</stepFunction>
+      <locations>
+        <sourceRange>
+          <path>src/xz/coder.h</path>
+          <start>
+            <line>49</line>
+            <column>17</column>
+          </start>
+          <end>
+            <line>49</line>
+            <column>30</column>
+          </end>
+        </sourceRange>
+      </locations>
     </configurationOption>
   </numericOptions>
   <booleanConstraints>
```

Comment:
I think you forgot to remove the Boolean constraints in line 1548-1550 (because you removed the LZMA1, LZMA2 and Delta feature):

```
<constraint>!lzma | LZMA1</constraint>
<constraint>!xz | !LZMA1</constraint>
<constraint>!FlushTO | LZMA2 | Delta</constraint>
```